PendingReleaseNotes

```diff
@@ -83,6 +83,23 @@
   redistribution. In contrast, a stretch rule with a single-take configuration
   will not cause any data movement during the upgrade process.
 
+* CephFS: Command "ceph fs volume create" now allows users to pass metadata
```

Comment:
* CephFS: The "ceph fs volume create" command now allows users to pass metadata

Reply:
Fixed.